### ISSUE TITLE
Clearly show multi-line items belong to the tree

### ISSUE THREAD

**Issue by epage** [softprops/treeline#4]
`assert_cmd` uses `predicates` for its assertions and sometimes we are asserting on multi-line stdout and the output can be hard to parse, see https://github.com/assert-rs/assert_cmd/issues/121

One idea we are exploring for this in predicates (see https://github.com/assert-rs/predicates-rs/issues/106) is that we indent the entire block o content to the same level, ie under the tree.  Ideally the branch would then flow within that indentation to the next item.  This would require support from treeline to do.

**Comment by softprops** [comment]
Thanks for posting. This makes sense to me

**Comment by epage** [comment]
Just want to check for understanding if you were expecting to go and implement this or if this was a greenlight for me.

**Comment by softprops** [comment]
Have at it!
